healdata.org/portal/gitops.json

```diff
@@ -122,7 +122,8 @@
     "discovery": true,
     "discoveryUseAggMDS": true,
     "studyRegistration": true,
-    "workspaceRegistration": true
+    "workspaceRegistration": true,
+    "workspaceTokenServiceRefreshTokenAtLogin": true
```

Comment:
```suggestion
    "workspaceTokenServiceRefreshTokenAtLogin": true,
    "discoveryUseAggWTS": true
```

Reply:
Thank you for the catch @mfshao , I have added the feature flag in the latest commit